test/quick_test_helper.rb

```diff
@@ -1,7 +1,4 @@
-if ENV['RAILS_ENV']
-  def rails_require(*)
-  end
-else
+if ENV['RAILS_ENV'] == "test"
```

Comment:
It prints "test".   Note that it's defined in `test/test_helper.rb`, so I wonder if there's some randomness in file load order?